lib/particular-calendars/france.coutances.ts

```diff
@@ -0,0 +1,244 @@
+
+    // src: mr_fr_1982_ed2_coutances
+    dedication_of_the_cathedral_of_coutances_france: {
+      dateDef: { month: 7, date: 12 },
+      precedence: Precedences.ProperFeast_DedicationOfTheCathedralChurch_8b,
+    },
+
+    // src: mr_fr_1982_ed2_coutances
+    helier_of_jersey_martyr: {
+      dateDef: { month: 7, date: 16 },
+      precedence: Precedences.OptionalMemorial_12,
+    },
+
+    // src: mr_fr_1982_ed2_coutances
+    marie_madeleine_postel_virgin: {
+      dateDef: { month: 7, date: 17 },
+      precedence: Precedences.ProperMemorial_11b,
+    },
+
+    // src: mr_fr_1982_ed2_coutances
+    clair_of_normandy_priest: {
+      dateDef: { month: 7, date: 18 },
+      precedence: Precedences.OptionalMemorial_12,
+    },
+
+    // src: mr_fr_1982_ed2_coutances
+    john_eudes_priest: {},
+
+    // src: mr_fr_1982_ed2_coutances
+    jacques_francois_lefranc_martyr_and_martyrs_of_the_revolution: {
+      dateDef: { month: 9, date: 2 },
+      precedence: Precedences.ProperMemorial_11b,
+      martyrology: ['jacques_francois_lefranc_martyr', { id: 'companions_martyrs', count: 23 }],
+    },
+
+    // src: mr_fr_1982_ed2_coutances
+    omer_audomar_bishop: {
+      dateDef: { month: 9, date: 9 },
+      precedence: Precedences.OptionalMemorial_12,
+    },
+
+    // src: mr_fr_1982_ed2_coutances
+    aubert_of_avranches_bishop: {
+      dateDef: { month: 9, date: 10 },
+      precedence: Precedences.ProperMemorial_11b,
+    },
+
+    // src: mr_fr_1982_ed2_coutances
+    floscel_martyr: {
+      dateDef: { month: 9, date: 17 },
+      precedence: Precedences.OptionalMemorial_12,
+    },
+
+    // src: mr_fr_1982_ed2_coutances
+    senerius_bishop: {
+      dateDef: { month: 9, date: 18 },
+      precedence: Precedences.OptionalMemorial_12,
+    },
+
+    // src: mr_fr_1982_ed2_coutances
+    vitalis_of_savigny_bishop: {
+      dateDef: { month: 9, date: 19 },
+      precedence: Precedences.ProperMemorial_11b,
+    },
+
+    // src: mr_fr_1982_ed2_coutances
+    laud_of_coutances_bishop: {
+      customLocaleId: 'laud_of_coutances_bishop_patron_of_the_diocese_of_coutances',
+      dateDef: { month: 9, date: 22 },
+      precedence: Precedences.ProperFeast_PrincipalPatronOfADiocese_8a,
+      titles: { append: [PatronTitle.PatronOfTheDiocese] },
+    },
+
+    // src: mr_fr_1982_ed2_coutances
+    john_de_brebeuf_isaac_jogues_priests_and_companions_martyrs: {
+      customLocaleId: 'john_de_brebeuf_priest_and_companions_martyrs',
+      dateDef: { month: 9, date: 25 },
+      precedence: Precedences.ProperMemorial_11b,
+      martyrology: ['john_de_brebeuf_priest', 'companions_martyrs'],
+    },
+
+    // src: mr_fr_1982_ed2_coutances
+    all_holy_bishops_of_the_diocese_of_coutances: {
+      dateDef: { month: 10, date: 12 },
+      precedence: Precedences.ProperMemorial_11b,
+    },
+
+    // src: https://www.diocese50.fr/les-paroisses-et-les-doyennes/notre-dame-coutances/les-saints-du-diocese/bienheureux-pierre-adrien-toulorge-martyr
```

Comment:
Sourcing all these additional celebrations not found in `mr_fr_1982_ed2_coutances` (due to being inscribed after its publishing) is quite strange to me, as the diocese50.fr website does not contain any celebration metadata (like the celebration date or rank), at least not on the referenced page.

It is a nice source for `fr.ts` though, however, I would not use it as a celebration inscription source.

@emagnier, could you provide us a better source please? :pray:

Reply:
> as the diocese50.fr website does not contain any celebration metadata (like the celebration date or rank)

Yes, this information is indeed provided on the page dedicated to each of these celebrations.  
For example, for this one: https://www.diocese50.fr/les-paroisses-et-les-doyennes/notre-dame-coutances/les-saints-du-diocese/bienheureuse-marthe-le-bouteiller-vierge
`Fêtée le 18 mars, mémoire facultative` -> 03-18, optional memorial.

I worked with the sources I had on hand. I will try again to obtain new sources, but I can't promise anything in the short term.

Reply:
Oh, sorry, I missed the celebration date and rank, sorry. Then, there is no problem with using diocese50.fr as a third-party source.